### PR TITLE
Add Mediawiki Hook Demo

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: apps/mediawiki/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+`oc new-project mediawiki`  
+`oc create -f mariadb.yml`  
+`oc create -f mediawiki-route.yml`
+  
+Update the `MEDIAWIKI_SITE_SERVER` value in `mediawiki.yml` with the results from running:  
+`oc get route -n mediawiki mediawiki -o go-template='{{printf "%s\n" .spec.host}}'`   
+  
+`oc create -f mediawiki.yml`
+
+When done setup CAM if you have not already. Run a migration and add a postRestore hook uploading `mediawiki-playbook.yml` as the playbook to run. Complete plan configuration and run a migration of the namespace.
```
Comment (rayfordj) [inline apps/mediawiki/README.md:14]: ~CAM~ MTC

**Review by rayfordj — CHANGES_REQUESTED** [review]

File: apps/mediawiki/README.md (hunk 1 above)
Comment (rayfordj) [reply apps/mediawiki/README.md]: Correction; upstream so: ~CAM~ -> Crane.

Comment (jmontleon) [reply apps/mediawiki/README.md]: I put `Crane or MTC`. It will work with either...

**Review by rayfordj — DISMISSED** [review]
LGTM

**Review by eriknelson — APPROVED** [review]

**Pull request merged**